docs/modules/user-guide/partials/proc_refering-to-a-parent-devfile-in-a-devfile.adoc

```diff
@@ -2,17 +2,22 @@
 = Refering to a parent devfile in a devfile
 
 [role="_abstract"]
-It is possible to refer to another devfile as a parent to a given devfile. If a parent is specified, the devfile will inherit all behaviour from the parent. Part of the content of the parent can be overridden in the child devfile. That allows refering a devfile (the parent) from a distinct devfile and makes it possible to reuse in multiple devfiles the same parent (the stack).
+This section describes how to designate a parent devfile to a given devifle. If you designate a parent devfile, the given devfile inherits all its behavior from its parent.  Still, you can use the child devfile to override certain content from the parent devfile. If you override the correct content, you can reuse the same parent devfile in multiple other devfiles. If you do reuse a parent devfile, the parent turns into a stack that is used in multiple other devfiles.
```

Comment:
Typo: "devifle"

I actually think the original opening sentence is a decent introduction before going into more detail. "This section describes..." feels superfluous.

Reply:
I think we wanted to start each abstract with "this section describes..." at least until we have a more user centric abstract written. 

Do you think it's fine as is? Or should I change the first sentence for all 8 docs?

Reply:
It's probably fine as-is for now, but I do think we should come back later and reevaluate how we're writing each abstract (if that makes sense).

Primarily, I'm concerned with the fact that this feels like fluff. Describing what a section is about to say is unnecessary if it's written properly in the first place, and I feel like the original first sentence of this section did so decently well. Why change it in that case? You're working against the future work across all of the abstracts.

Reply:
Cool, let's address this concern in #402: https://github.com/devfile/api/issues/402